src/index.html

```diff
@@ -9,6 +9,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__link">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="list__element">
+            <a href="#" class="list__link is-active">Apple</a>
+          </li>
+          <li class="list__element">
+            <a href="#" class="list__link">Samsung</a>
+          </li>
+          <li class="list__element">
+            <a href="#" class="list__link">Smartphones</a>
+          </li>
```

Comment:
Using spaces between child elements will increase readability

```suggestion
        <ul class="navigation__list">
          <li class="list__element">
            <a href="#" class="list__link is-active">Apple</a>
          </li>
          
          <li class="list__element">
            <a href="#" class="list__link">Samsung</a>
          </li>
          
          <li class="list__element">
            <a href="#" class="list__link">Smartphones</a>
          </li>
```